testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -124,6 +124,32 @@ impl EmitJobRequest {
             },
         }
     }
+
+    pub fn fixed_tps_params(instance_count: usize, tps: u64) -> (usize, u64) {
+        let tps = if tps < 1 { 1 } else { tps };
```

Comment:
I don't think we need this check?
`--tps 0` is clearly a error, we can just panic there